tools/Dockerfile

```diff
@@ -7,14 +7,17 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN microdnf update && \
     microdnf install java-11-openjdk-headless --nodocs && \
     microdnf install tar gzip zip shadow-utils curl openssl && \
+    microdnf install python3-devel python3-pip python3-setuptools && \
     microdnf clean all
 
 # The bcrypt tool is needed by NiFi to locally encrypt the admin password that is mounted as a secret in cleartext
 COPY tools/bin/stackable-bcrypt-1.0-SNAPSHOT-jar-with-dependencies.jar /bin/stackable-bcrypt.jar
+# add all python scripts
+COPY tools/python /stackable/python
 
-RUN groupadd -r stackable --gid=1000 && \
+RUN pip3 install -r /stackable/python/requirements.txt && \
```

Comment:
Should we not try to keep this in a virtualenv? In case we later add more scripts with different/conflicting dependencies that might make more sense, no?
Then again, virtualenvs in containers must be some sort of antipattern..

Reply:
I would make more sense, but i think we can tackle that once we have more scripts.